extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -138,7 +139,20 @@ export class AmpAdXOriginIframeHandler {
         this.adResponsePromise_,
         'timeout waiting for ad response').catch(e => {
           this.noContent_();
-          user().warn('AMP-AD', e);
+          getAdCid(this.baseInstance_).then(cid => {
+            const tagType = this.baseInstance_.element.getAttribute('type');
+            const TAG = `AMP-AD-${tagType}`;
+            dev().error(TAG, JSON.stringify({
+              'm': e instanceof Error ? e.message : e,
+              'type': tagType,
+              'correlator': this.baseInstance_.win.ampAdPageCorrelator ||
+                  null,
+              'slotId': this.baseInstance_
+                  .element.getAttribute('data-amp-slot-index'),
+              'clientId': cid,
```

Comment:
@cramforce, @tdrl, @glevitzky 
What about a smaller set of data sent in error report? Even just including the tagType would potentially signal existing problems by raw counts and alert on new problems that occur. Even something this simple would have caught the recent rendering issue in GPT Light when we ramped up the traffic fraction.